GVRf/Framework/src/org/gearvrf/GVRHybridObject.java

```diff
@@ -153,6 +154,17 @@ public long getNative() {
         return mNativePointer;
     }
 
+    public static long[] getNativePtrArray(Collection<? extends GVRHybridObject> objects) {
```

Comment:
I am a little bit dubious about treating public member as strictly end-user API. Java has no nested packages, package-private does not always provide adequate visibility to framework classes themselves -- unless all framework classes are in one package. This leads to disorganization in the framework. The gain here would lead to liabilities as the framework grows.